locals.tf

```
@@ -42,15 +42,19 @@ locals {
   admin_username = "jenkins-infra-team"
 
   kubernetes_versions = {
-    "privatek8s"           = "1.27.9"
-    "publick8s"            = "1.27.9"
-    "cijenkinsio_agents_1" = "1.27.9"
+    "privatek8s"                = "1.27.9"
+    "infracijenkinsio_agents_1" = "1.27.9"
+    "publick8s"                 = "1.27.9"
+    "cijenkinsio_agents_1"      = "1.27.9"
   }
   ci_jenkins_io_fqdn              = "ci.jenkins.io"
-  ci_jenkins_io_agents_1_pod_cidr = "10.100.0.0/14"
+  ci_jenkins_io_agents_1_pod_cidr = "10.100.0.0/14" # 10.100.0.1 - 10.103.255.255
```

```suggestion
  cijenkinsio_agents_1_compute_zones      = [1]
  ci_jenkins_io_agents_1_pod_cidr = "10.100.0.0/14" # 10.100.0.1 - 10.103.255.255
```
nit: moved from below.